src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -550,6 +550,20 @@ public static void CopyTo_Invalid()
         [InlineData("Goodbye", 0, "Hello", 0, 5, StringComparison.OrdinalIgnoreCase, -1)]
         [InlineData("HELLO", 2, "hello", 2, 3, StringComparison.OrdinalIgnoreCase, 0)]
         [InlineData("Hello", 2, "Goodbye", 2, 3, StringComparison.OrdinalIgnoreCase, -1)]
+        [InlineData("A", 0, "x", 0, 1, StringComparison.OrdinalIgnoreCase, -1)]
```

Comment:
Yeah, that'd be a good thing to add. I already merged this, but it can be done separately.